### DISCUSSION TITLE
Is there a cadence for releases on the tock-registers crate?

### DISCUSSION THREAD

**Discussion by andreeaflorescu** [tock/tock#3691] [Q&A] [answered]
Hey, I just noticed that the last release of tock-registers is from September 2022. How often do releases happen?

**Comment by ppannuto** [comment, marked as answer]
There's not a formal release cadence, rather we tend to release on-demand when salient features are added, fixes needed by downstream, etc.

Looks like there's only two commits of note since the last release:

 - 480ee65139f1e51c149236a1b3e47cd61832ac80: Rename matches_any() to any_matching_bits_set(), implement matches_any()
 - 83d1665c61c27525ccdaa8262429264ef041b317: derive Debug on Value

Happy to push a new release; I'm guessing given your recent PR you'd like the `Debug` fix pushed out?

I'll work on putting together a release now.

**Comment by andreeaflorescu** [comment]
Hey @ppannuto thanks for the quick reply. Yap, I was asking about the derive Debug so I can simplify some code we're having. Thanks!

**Reply by ppannuto** [reply]
https://crates.io/crates/tock-registers :)
